src/dotnet-libman/Commands/InstallCommand.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.CommandLineUtils;
+using Microsoft.Web.LibraryManager.Contracts;
+
+namespace Microsoft.Web.LibraryManager.Tools.Commands
+{
+    internal class InstallCommand : BaseCommand
+    {
+        public InstallCommand(IHostEnvironment hostEnvironment, bool throwOnUnexpectedArg = true)
+            : base(throwOnUnexpectedArg, "install", Resources.InstallCommandDesc, hostEnvironment)
+        {
+        }
+
+        public CommandArgument LibraryId { get; private set; }
+        public CommandOption Provider { get; private set; }
+        public CommandOption Destination { get; set; }
+        public CommandOption Files { get; set; }
+
+        public override BaseCommand Configure(CommandLineApplication parent)
+        {
+            base.Configure(parent);
+
+            LibraryId = Argument("libraryId", Resources.InstallCommandLibraryIdArgumentDesc, multipleValues: false);
+            Provider = Option("--provider|-p", Resources.ProviderOptionDesc, CommandOptionType.SingleValue);
+            Destination = Option("--destination|-d", Resources.DestinationOptionDesc, CommandOptionType.SingleValue);
+            Files = Option("--files", Resources.FilesOptionDesc, CommandOptionType.MultipleValue);
+
+            return this;
+        }
+
+        protected override async Task<int> ExecuteInternalAsync()
+        {
+            Manifest manifest = await GetManifestAsync(createIfNotExists: true);
+
+            ValidateParameters(manifest);
+
+            List<string> files = Files.HasValue() ? Files.Values : null;
+
+            ILibraryInstallationState library = new LibraryInstallationState()
+            {
+                LibraryId = LibraryId.Value,
```

Comment:
Not yet. 
Right now it will just add 
```json
{
  "libraryId": "jquery"
}
```
and then restore would fail for cdnjs. I am going to look into what we can do here to either 
1. Fail early and not let the library be added to the json file.
2. Figure out the latest version and use that to add the library.